### ISSUE TITLE
[SAGE] Home Page

### ISSUE THREAD

**Issue by devangm** [TAMULib/SAGE#563]
**URL**: https://library.tamu.edu/discovery/
**Violation**: AA: Contrast (Minimum) 
**Issues**: Very low contrast 
**Notes**: White text on buttons have vey low contrast with background

**URL**:https://library.tamu.edu/discovery/ 
**Notes**: NVDA - After navigating out of header, there is a random clickable menu being called out. The profile dropdown is also just called as 'clickable' instead of profile menu

**URL**https://library.tamu.edu/discovery/ 
**Notes**: Learn More' links to github

**Comment by qtamu** [comment]
As of right now there is not clear direction on what URL the "Learn More" button is supposed to point to.
